mbed_app.json

```diff
@@ -40,6 +40,13 @@
             ],
             "target.components_add": ["STMOD_CELLULAR"],
             "stmod_cellular.provide-default": "true"
+        },
+        "EP_AGORA": {
+            "target.features_add"           : ["LWIP"],
+            "target.features_remove"        : ["BLE"],
+            "target.extra_labels_remove"    : ["CORDIO", "CORDIO_LL", "NORDIC_CORDIO"],
+            "target.extra_labels_add"       : ["SOFTDEVICE_NONE"],
```

Comment:
Our target should be soft device none by default anyways right?

Reply:
Also wondering why Cordio & bluetooth would end up compiled in if none of its API's are used by the example.

do we need to override remove these features from the target?

Reply:
Looks like you're right in that we don't need to add the `SOFTDEVICE_NONE` label as it's already there. Here's a snippet from the `targets.json` file for the `MCU_NRF52840` target (which the `EP_AGORA` target inherits from):
```
        "extra_labels": [
            "NORDIC",
            "NRF5x",
            "NRF52",
            "SDK_15_0",
            "CORDIO",
            "CORDIO_LL",
            "SOFTDEVICE_NONE",
            "NORDIC_CORDIO"
        ],
```
So, I guess you could argue we don't need to remove the `CORDIO` labels if we're already removing the `BLE` feature, but it doesn't hurt.

Reply:
@40Grit Are you suggesting that we shouldn't need to remove the `BLE` feature or the `CORDIO` labels since we never call any BLE functions (and therefore it won't get compiled in)?

Reply:
I'm hoping intuitively that is the case. But I know better than to assume.

Especially considering what the tools docs say about FEATURE labeled sources.

Reply:
I retried this example without the following lines, and it still works:
```
            "target.features_remove"        : ["BLE"],
            "target.extra_labels_remove"    : ["CORDIO", "CORDIO_LL", "NORDIC_CORDIO"],
            "target.extra_labels_add"       : ["SOFTDEVICE_NONE"],
```

So, I think we can safely take them out, and make it a more simple change.